prow/config.yaml

```diff
@@ -1063,3 +1063,39 @@ periodics:
     - hostPath:
         path: /mnt/disks/ssd0
       name: cache-ssd
+
+- name: ci-perf-tests-e2e-gce-clusterloader
+  interval: 30m
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170418-c08e1094
+      args:
+      - "--repo=k8s.io/kubernetes=master"
+      - "--repo=k8s.io/perf-tests=master"
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name:  JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-private
+      - name:  JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-public
```

Comment:
you might not need those ssh entries. But if you do, you'll also need to add an USER:prow entry.